src/components/DatePickerInput/DatePickerInput.js

```diff
@@ -147,21 +131,13 @@ export default class DatePickerInput extends Component {
 
     return (
       <div className={containerClasses}>
-        {componentsX ? (
-          <>
-            {label}
-            <div className={`${prefix}--date-picker-input__wrapper`}>
-              {input}
-              {datePickerIcon}
-            </div>
-          </>
-        ) : (
-          <>
-            {datePickerIcon}
-            {label}
+        <>
```

Comment:
we don't need the fragment here since it's no longer in a conditional expression right?

Reply:
Good catch @emyarod! Updated.